hadoop-ozone/dev-support/checks/_lib.sh

```diff
@@ -73,7 +73,7 @@ install_k3s() {
 }
 
 _install_k3s() {
-  curl -sfL https://get.k3s.io | sh -
+  curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION="v1.21.2+k3s1" sh -
```

Comment:
These changes to CI check library seem to be unrelated.

Reply:
This a known bug for kubernetes k3s that appeared a few weeks ago due to a version upgrade to k3s (resolved on master) that appears in the feature branch apache:HDDS-4440-s3-performance.  The added line to _lib.sh fixes the issue for the feature branch.

Reply:
These kinds of fixes should be brought into the feature branch via merge from `master`.

Reply:
Thanks @adoroszlai for the comment on bugs fixed on master that affect our feature branch.  Can you please apply it to the feature branch` apache:HDDS-4440-s3-performance` from the `master` through a merge?  I'll update our PR and re-run CI afterwords.

Reply:
Updated `HDDS-4440-s3-performance` from `master`: 3b4e2f58b35f1e2dbf553ea183063867f1fd4771.

Reply:
Thanks @adoroszlai ; updated commit with your update to the feature branch.